EmojiArt/Document/Views/EmojiArtDocumentView.swift

```diff
@@ -6,6 +6,7 @@ struct EmojiArtDocumentView: View {
     }
 
     @ObservedObject var documentViewModel: EmojiArtViewModel
+    @State private var selectedEmojis = Set<EmojiArtModel.Emoji>()
```

Comment:
I would do `selectedEmojis: Set<EmojiArtModel.Emoji> = .init()`, not required to change